x/tunnel/keeper/keeper_packet_ibc_hook.go

```diff
@@ -0,0 +1,73 @@
+package keeper
+
+import (
+	"time"
+
+	ibctransfertypes "github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/bandprotocol/chain/v3/x/tunnel/types"
+)
+
+// SendIBCHookPacket sends a packet to the destination chain using IBC Hook
+func (k Keeper) SendIBCHookPacket(
+	ctx sdk.Context,
+	route *types.IBCHookRoute,
+	packet types.Packet,
+	feePayer sdk.AccAddress,
+	interval uint64,
+) (types.PacketReceiptI, error) {
+	// create memo string for ibc transfer
+	pricePacket := types.NewTunnelPricesPacketData(packet.TunnelID, packet.Sequence, packet.Prices, packet.CreatedAt)
+	memoStr := types.NewIBCHookMemo(route.DestinationContractAddress, pricePacket).JSONString()
+
+	// mint coin to the fee payer
+	err := k.MintIBCHookCoinToAccount(ctx, packet.TunnelID, feePayer)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Can we make the function receive the amount instead of hardcoding it in the function?